app/main.py

```diff
@@ -18,2 +1,10 @@
 
-# write your code here
+sorted_variables = {
+    "mutable": [b, d],
+    "immutable": [a, c]}
+a = 5
+b = ["jakow"]
+c = "Hello, worrld"
+d = "orange"
+mutable: [b, d]
+immutable: [a, c]
```

Comment:
Similar to the previous line, this line is incorrectly formatted. If you are trying to define a dictionary or list, ensure you use the correct syntax.